starknet-foundry/crates/test-collector/src/project.rs

```diff
@@ -0,0 +1,81 @@
+// COPIED FROM CAIRO COMPILER
```

Comment:
If sourcing a file like that, it'd be useful if you provided the original path/lines